components/script/dom/element.rs

```diff
@@ -20,7 +20,7 @@ use dom::bindings::codegen::Bindings::HTMLTemplateElementBinding::HTMLTemplateEl
 use dom::bindings::codegen::Bindings::NamedNodeMapBinding::NamedNodeMapMethods;
 use dom::bindings::codegen::Bindings::NodeBinding::NodeMethods;
 use dom::bindings::codegen::InheritTypes::{CharacterDataCast, DocumentDerived, ElementCast};
-use dom::bindings::codegen::InheritTypes::{ElementDerived, EventTargetCast, HTMLAnchorElementCast};
+use dom::bindings::codegen::InheritTypes::{ElementDerived, EventTargetCast, HTMLAnchorElementCast, HTMLFieldSetElementDerived, HTMLLegendElementDerived, HTMLOptGroupElementDerived};
```

Comment:
*All* of the imports need to be sorted, so check out the next line and notice how those fall alphabetically before some of the imports on this line. Let me know if you need help